src/main/java/org/influxdb/dto/Point.java

```diff
@@ -196,6 +225,104 @@ public boolean hasFields() {
       return !fields.isEmpty();
     }
 
+    /**
+     * Adds field map from object by reflection using {@link org.influxdb.annotation.Column}
+     * annotation.
+     *
+     * @param pojo POJO Object with annotation {@link org.influxdb.annotation.Column} on fields
+     * @return the Builder instance
+     */
+    public Builder addFieldsFromPOJO(final Object pojo) {
+
+      Class<? extends Object> clazz = pojo.getClass();
+
+      for (Field field : clazz.getDeclaredFields()) {
+
+        Column column = field.getAnnotation(Column.class);
+
+        if (column == null) {
+          continue;
+        }
+
+        String fieldName = column.name();
+
+        if (isPublic(field)) {
```

Comment:
I don't think it matters if the field is public or not: you should get the field value anyway as it was annotated with (at)Column annotation.

Reply:
I've tested it, if it's private, it cannot be accessed. 
I think I can set `accessible` to true. I'll test it and update the pull request.